api/models/post.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require("sequelize");
+const sequelize = require("../db/db.js");
+const User = require("./User.js");
+
+const Post = sequelize.define("posts", {
+  id: {
+    type: Sequelize.INTEGER(11),
+    allowNull: false,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+
+  topic: Sequelize.STRING(250),
+  discription: Sequelize.STRING(400),
+  city: Sequelize.STRING(255),
+  user_id: Sequelize.INTEGER(11),
+  published: Sequelize.BOOLEAN,
+  // Timestamps
```

Comment:
how we will use published?I don't get it so can you explain it to me ?

Reply:
it is totally optional 
1. i want it to go true when a user posts a post
2. then we can use it to show all the posts that are published